SIT/smoke/01_basic_kvp/cdm.sh

```diff
@@ -10,7 +10,7 @@ EOF
 spark-submit \
   --properties-file /smoke/01_basic_kvp/migrate.properties \
   --master "local[*]" \
-  --class datastax.astra.migrate.Migrate /local/cassandra-data-migrator.jar 
+  --class datastax.cdm.job.Migrate /local/cassandra-data-migrator.jar
```

Comment:
shouldn't this be `com.datastax.cdm.job.Migrate` as the class is [here](https://github.com/datastax/cassandra-data-migrator/blob/main/src/main/scala/com/datastax/cdm/job/Migrate.scala)?

Reply:
@msmygit  It should be --class com.datastax.cdm.job cc: @mieslep , will raise a separate JIRA ticket to resolve the refactoring issues  synk security check.